### ISSUE TITLE
Create redirect proxy for urls from Federalist to Pages

### ISSUE THREAD

**Issue by apburnes** [cloud-gov/pages-proxy#173]
In order to support the Pages transition, add a redirect proxy from Federalist to Pages

Enabled by

- https://github.com/cloud-gov/pages-proxy/pull/174
- https://github.com/cloud-gov/pages-proxy/pull/175
- https://github.com/cloud-gov/pages-proxy/pull/176

Waiting on https://github.com/cloud-gov/product/issues/2092 to be closed

## Acceptance Criteria
* [x] Create new issue to send out deprecation notice to users of the old preview urls
* [x] Given a Federalist site preview url, redirect `<site-url>.app.cloud.gov` to the cloud.gov Pages proxy `<site-url>.sites.pages.cloud.gov`

--- 

## Security considerations
Need for Pages migration

## Implementation sketch
[links to background notes, sketches, and/or relevant documentation
* [x] Add an NGINX config to redirect trafic to `*.sites.pages.cloud.gov`
* [x] Deploy via the `federlist-proxy` pipeline
* [x] Update the current CF app `federalist-proxy` with the new NGINX config in order to keep the site routes mapped to the current proxy

**Comment by apburnes** [comment]
Proxies created using https://github.com/cloud-gov/pages-proxy#short-term-site-redirects directions.
